src/components/Shared/NavBar.vue

```diff
@@ -2,101 +2,107 @@
 <template>
   <nav>
     <v-app-bar app class="background">
-      <v-btn icon router to="/" color="white" class="hidden-xs-only ml-3">
+      <v-app-bar-nav-icon
+        v-if="mobileView"
+        @click.stop="drawer = !drawer"
+      ></v-app-bar-nav-icon>
+      <v-btn icon router to="/" color="white" class="ml-3">
         <v-icon x-large>$vuetify.icons.cp-logo</v-icon>
       </v-btn>
+
       <v-toolbar-title class="text-uppercase black--text">
         <span class="font-weight-light display-1">WISP</span>
       </v-toolbar-title>
 
       <v-spacer></v-spacer>
 
-      <v-btn
-        text
-        tile
-        router
-        to="/"
-        min-height="115%"
-        class="offset-y primary--text"
-        >home</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/dashboard"
-        min-height="115%"
-        v-if="$store.state.isLoggedIn"
-        class="offset-y primary--text"
-        >dashboard</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/problemSets"
-        min-height="115%"
-        v-if="$store.state.isLoggedIn"
-        class="offset-y primary--text"
-        >problem sets</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/problems"
-        min-height="115%"
-        v-if="$store.state.isLoggedIn"
-        class="offset-y primary--text"
-        >problems</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/admin"
-        min-height="115%"
-        v-if="$store.state.isLoggedIn && $store.state.user.role === 'ADMIN'"
-        class="offset-y primary--text"
-        >admin</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/about"
-        min-height="115%"
-        class="offset-y primary--text"
-        >about</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/login"
-        min-height="115%"
-        v-if="!$store.state.isLoggedIn"
-        class="offset-y primary--text"
-        >login</v-btn
-      >
-
-      <v-btn
-        text
-        tile
-        router
-        to="/signup"
-        min-height="115%"
-        v-if="!$store.state.isLoggedIn"
-        class="offset-y primary--text"
-        >register</v-btn
-      >
+      <v-item-group v-if="!mobileView">
```

Comment:
v-item-group might be causing problems with the background of you buttons:

before
![image](https://user-images.githubusercontent.com/41027939/82272562-f37d6d80-9948-11ea-8e66-ad599e401a5a.png)

after
![image](https://user-images.githubusercontent.com/41027939/82272520-d8126280-9948-11ea-8fcb-81739b81424d.png)

Also, it might be causing the buttons to clump up on small, non-mobile window sizes:

before:
![image](https://user-images.githubusercontent.com/41027939/82272640-2cb5dd80-9949-11ea-9bc2-259ca21605cf.png)

after:
![image](https://user-images.githubusercontent.com/41027939/82272625-21fb4880-9949-11ea-8673-6dedbabd1565.png)